src/style.css

```diff
@@ -0,0 +1,100 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+:root {
+  --main-text-color: #000;
+  --main-border-color: #e1e7ed;
+}
+
+.bar {
+  display: block;
+  box-sizing: border-box;
+  position: relative;
+  width: 100%;
+  border: 1px solid var(--main-border-color);
+  border-radius: 4px;
+  background: #fff;
+  cursor: pointer;
+  margin-top: 20px;
+  /* box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1); */
```

Comment:
The box-shadow property is commented out. If you want to apply the box-shadow, uncomment this line.